app-tracking-protection/vpn-impl/src/main/java/com/duckduckgo/mobile/android/vpn/ui/alwayson/AlwaysOnAlertDialogFragment.kt

```diff
@@ -32,15 +33,20 @@ import com.google.android.material.bottomsheet.BottomSheetBehavior
 import com.google.android.material.bottomsheet.BottomSheetDialog
 import com.google.android.material.bottomsheet.BottomSheetDialogFragment
 import dagger.android.support.AndroidSupportInjection
-import java.lang.ref.WeakReference
+import timber.log.Timber
 import javax.inject.Inject
 
+internal inline class DialogHeading(@IdRes val id: Int)
```

Comment:
why not pass a simple flag as an argument to the Fragment and use that to decide which string resources to display?

Reply:
To create the fragment I think I refer the separate functions rather than just a eg. `boolean` flag (hate boolean flags).
But then, I can pass an argument to the fragment and let the `bindViewElements()` method do the `if-else`.

Will do